include/gazebo_mavlink_interface.h

```diff
@@ -66,6 +66,7 @@
 #include <Odometry.pb.h>
 #include <MagneticField.pb.h>
 #include <Pressure.pb.h>
+#include "Wind.pb.h"
```

Comment:
Why is this one "" and all the other plugins <>? Aren't they all included the same way and it should be consistent?